File: app/src/main/java/com/example/reccardapp/TeamsDetailedActivity.java
package com.example.reccardapp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ImageView;
import android.widget.TextView;

public class TeamsDetailedActivity extends AppCompatActivity
{
   ImageView detail_Img;
   TextView detail_team_Name,detail_short_Name, detail_team_Description, detail_team_Members;
    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.teams_activity_main_detail);

        detail_Img=(ImageView)findViewById(R.id.detail_img);
        detail_team_Name=(TextView)findViewById(R.id.detail_team_name);
        detail_short_Name=(TextView)findViewById(R.id.detail_short_name);
        detail_team_Description=(TextView)findViewById(R.id.detail_team_description);
        detail_team_Members=(TextView)findViewById(R.id.detail_team_members);

        detail_Img.setImageResource(getIntent().getIntExtra("imagename",0));
        detail_team_Name.setText(getIntent().getStringExtra("teamName"));
        detail_short_Name.setText(getIntent().getStringExtra("shortName"));
        detail_team_Description.setText(getIntent().getStringExtra("teamDescription"));
        detail_team_Members.setText(getIntent().getStringExtra("teamMembers"));
    }
}

File: app/src/main/java/com/example/reccardapp/TeamsDetailedActivity.kt
package com.example.reccardapp

import android.os.Bundle
import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class TeamsDetailedActivity : AppCompatActivity() {
    var detail_Img: ImageView? = null
    var detail_team_Name: TextView? = null
    var detail_short_Name: TextView? = null
    var detail_team_Description: TextView? = null
    var detail_team_Members: TextView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.teams_activity_main_detail)
        detail_Img = findViewById<View>(R.id.detail_img) as ImageView
        detail_team_Name = findViewById<View>(R.id.detail_team_name) as TextView
        detail_short_Name = findViewById<View>(R.id.detail_short_name) as TextView
        detail_team_Description = findViewById<View>(R.id.detail_team_description) as TextView
        detail_team_Members = findViewById<View>(R.id.detail_team_members) as TextView
        detail_Img!!.setImageResource(intent.getIntExtra("imagename", 0))
        detail_team_Name!!.text = intent.getStringExtra("teamName")
        detail_short_Name!!.text = intent.getStringExtra("shortName")
        detail_team_Description!!.text = intent.getStringExtra("teamDescription")
        detail_team_Members!!.text = intent.getStringExtra("teamMembers")
    }
}